daemon/src/oracle.rs

```diff
@@ -69,7 +68,7 @@ where
         }
 
         Self {
-            latest_announcements: None,
+            latest_announcements: HashMap::new(),
```

Comment:
:question: Is it still worth regularly caching latest announcements now that we are not pushing them out to the CFD actor? We could simply go to the oracle with the `event_id`(s) that we constructed/received?

Reply:
The idea was to decouple IO errors from the actual contract-setup.